source/NVDAObjects/window/scintilla.py

```diff
@@ -193,9 +193,9 @@ def _getLineOffsets(self,offset):
 			# Lines in Scintilla refer to document lines, not wrapped lines.
 			# There's no way to retrieve wrapped lines, so use screen coordinates.
 			y=watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,offset)
-			top,left,width,height=self.obj.location
-			start = self._getOffsetFromPoint(0,y)
-			end=self._getOffsetFromPoint(width,y)
+			left,top,width,height=self.obj.location

```

Comment:
note that you can make use of locationHelper here, so you do not have to extract the tuple.

```suggestion
			location = self.obj.location
```

Reply:
This way might be less error prompt and is a bit clearer.